tools/admin/src/main/scala/whisk/core/database/UserCommand.scala

```diff
@@ -164,14 +172,23 @@ class UserCommand extends Subcommand("user") with WhiskCommand {
     authStore
       .get[ExtendedAuth](DocInfo(create.subject()))
       .flatMap { auth =>
+        val nsToUpdate = create.desiredNamespace(authKey).name
+        var newNS = auth.namespaces.filter(_.namespace.name != nsToUpdate)
```

Comment:
Avoid use of `var`. Instead name it as `existingNS` and then later just add to it and get `newNS`